site/src/main/paradox/Scio-Unit-Tests.md

```diff
@@ -98,11 +98,27 @@ Scio provides convenience methods for constructing `PTransformOverride`s in the
 Continuing the example above, `TransformOverride.ofAsyncLookup` can be used to map static mock data into the expected output format
 for the transform, here `KV[Int, BaseAsyncLookupDoFn.Try[String]]`.
 
-@@snip [JobTestTest.scala](/scio-test/src/test/scala/com/spotify/scio/testing/JobTestTest.scala) { #JobTestTest_example_2 }
+@@snip [JobTestTest.scala](/scio-test/src/test/scala/com/spotify/scio/testing/JobTestTest.scala) { #JobTestTest_example_2_1 }
+
+or use a T to Iterable[U] mapping instead, if it is more convenient:
```

Comment:
Per our discussion on Friday this isn't correct; it's not a matter of convenience it's the nature of the transform that dictates which you use.

Reply:
In this case it is convenience, because the original transform being overridden is 1-to-1. The example at line 115 is for the the case you have to use this override. Wdyt?

Reply:
found the actual misassumption in my impl. working to fix it. thanks for precise review.